src/Api/ApiResponse.php

```diff
@@ -36,19 +36,4 @@ public function withError (?string $error) : self
 		$this->error = $error;
 		return $this;
 	}
-
-
-	/**
-	 */
-	public function toArray () : array
```

Comment:
This is technically a BC break, but we have no consumer for that. That would break anyway as soon as we allow translatables.